CMakeLists.txt

```diff
@@ -39,17 +40,22 @@ set (CMAKE_INSTALL_FULL_PKGLIBEXECDIR "${CMAKE_INSTALL_FULL_LIBEXECDIR}/${CMAKE_
 find_package (PkgConfig REQUIRED)
 include (FindPkgConfig)
 
-pkg_check_modules(GLIB REQUIRED
+set (
+    GLIB
     glib-2.0>=2.36
     gio-2.0>=2.36
 )
-include_directories (${GLIB_INCLUDE_DIRS})
 
-pkg_check_modules(
-  URLDISPATCHER
-  lomiri-url-dispatcher>=0
-)
-include_directories(${URLDISPATCHER_INCLUDE_DIRS})
+if (ENABLE_LOMIRI_FEATURES)
+    list (
+        APPEND
+        GLIB
+        lomiri-url-dispatcher>=0
+    )
```

Comment:
..., because here it becomes more than GLIB...

Reply:
This breaks a good readability and makes people scratch there head. I'd use DEPS instead.